### PR TITLE
Task: converting from one number system to another was added

### PR DESCRIPTION
**Pull request by ZheltokDina**

### CODE REVIEW

File: taskDEV3/EntryPoint.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{
+  /// <summary>
+  /// The entry point of the programm
+  /// </summary>
+  class EntryPoint
+  {
+    static void Main(string[] args)
+    {
+      try
+      {
```
Comment (IgorKheidorov) [inline taskDEV3/EntryPoint.cs]: Why int? Why not BigInteger? Where is check for number of parameters?

File: taskDEV3/NumberConverter.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    /// </summary>
+    /// <param name="numberInDecimalForm">the number for transformation </param>
+    /// <param name="radix">radix in which necessary to transform</param>
+    public NumberConverter(int numberInDecimalForm, int radix)
+    {
+      if(radix < 2 || radix > 20 || numberInDecimalForm <= 0)
+      {
+        throw new ArgumentOutOfRangeException();
+      }
+      if(numberInDecimalForm <= 0)
```
Comment (IgorKheidorov) [inline taskDEV3/NumberConverter.cs]: Code style violation

File: taskDEV3/NumberConverter.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    /// </summary>
+    /// <returns> convertible number</returns>
+    public string ConvertNumberToNewRadix()
+    {
+      int bufferNumber = NumberInDecimalForm;
+      StringBuilder numberInNewRadix = new StringBuilder();
+      do
+      {
+        bufferNumber = NumberInDecimalForm / Radix;
+        int remainder = NumberInDecimalForm % Radix;
```
Comment (IgorKheidorov) [inline taskDEV3/NumberConverter.cs]: Do you check that it works properly? This is bug, '1' is not 0x1. Check

File: taskDEV3/EntryPoint.cs [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        BigInteger numberInDecimalForm = BigInteger.Parse(args[0]);
+        int radix = int.Parse(args[1]);
+        NumberConverter converter = new NumberConverter(numberInDecimalForm, radix);        
+        string convertibleNumber = converter.ConvertNumberToNewRadix();
```
Comment (AlexanderSoroka) [inline taskDEV3/EntryPoint.cs]: useless variable

File: taskDEV3/EntryPoint.cs [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  /// The entry point of the programm
+  /// </summary>
+  class EntryPoint
+  {
+    static void Main(string[] args)
+    {
+      try
+      {
+        BigInteger numberInDecimalForm = BigInteger.Parse(args[0]);
+        int radix = int.Parse(args[1]);
```
Comment (AlexanderSoroka) [inline taskDEV3/EntryPoint.cs]: useless variable

File: taskDEV3/EntryPoint.cs [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+        NumberConverter converter = new NumberConverter(numberInDecimalForm, radix);        
+        string convertibleNumber = converter.ConvertNumberToNewRadix();
+        Console.WriteLine($"Convertible number is { convertibleNumber}");
+      }
+      catch (FormatException ex)
+      {
+        Console.WriteLine(ex.Message);
```
Comment (AlexanderSoroka) [inline taskDEV3/EntryPoint.cs]: Do not make specific exception 'catch' block with the same processing

File: taskDEV3/NumberConverter.cs [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+using System.Numerics;
+
+namespace taskDEV3
+{
+  /// <summary>
+  /// This class is used for converting from decimal number system to another
+  /// </summary>
+  class NumberConverter
+  {
+    private const int NUMBER_TO_CHAR_START_POINT = 55;
```
Comment (AlexanderSoroka) [inline taskDEV3/NumberConverter.cs]: bad name

**Pull request merged**